dashboards/resource_lakeview_dashboard.go

```diff
@@ -0,0 +1,233 @@
+package dashboards
+
+import (
+	"bufio"
+	"context"
+	"crypto/md5"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"time"
+
+	"github.com/databricks/databricks-sdk-go/service/dashboards"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// Reads the file content from a given path
+func readFileContent(source string) ([]byte, error) {
+	log.Printf("[INFO] Reading %s", source)
+	f, err := os.Open(source)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+	reader := bufio.NewReader(f)
+	return io.ReadAll(reader)
+}
```

Comment:
I think we have sharable helper function for this already? cc @hectorcast-db

Reply:
https://github.com/databricks/terraform-provider-databricks/blob/main/workspace/file_resource.go#L22
I think we can make it public

Reply:
I took inspiration from that function only. But, is it not considered good to have a new function in the code. (The function I used is bit different and some of the things in old function are depricated)

Reply:
We can update the deprecated code. The deprecation message tells you to just replace the line with your line. These 2 functions do the same. Reusing the code keeps it easier in the future (less code, less maintenance).